eng/Versions.props

```diff
@@ -6,16 +6,26 @@
     Roslyn version
   -->
   <PropertyGroup>
+    <!-- All packages produced by the Roslyn official build should be suffixed with '-vs' to distinguish them from VMR produced packages -->
+    <_RoslynPackageSuffix Condition="'$(DotNetBuildFromVMR)' != 'true'">-vs</_RoslynPackageSuffix>
```

Comment:
And the -vs versions are still what are being inserted into VS? Do we have any cases where other packages are produced in the VS tree, and will end up with dependencies depending on these VS versions?

Reply:
> And the -vs versions are still what are being inserted into VS?

Yes

> Do we have any cases where other packages are produced in the VS tree, and will end up with dependencies depending on these VS versions?

Not public packages as far as I have been able to tell.  If they are, they would switch to reference the VMR produced version.

Reply:
> Not public packages as far as I have been able to tell. If they are, they would switch to reference the VMR produced version.

Is that something we somehow need to add validation to ensure? Because I imagine we'll discover that only after some `-vs` package got published and folks come to us wondering where it is.